teps/0001-tekton-enhancement-proposal-process.md

```diff
@@ -0,0 +1,471 @@
+[suggestions](https://github.com/tektoncd/community/blob/master/process.md#proposing-features)
+on proposing a feature. We rely on documents hosted on Google docs,
+without a standard template explaining the change. Once a proposal is
+done, via a design docs, it tends to be hard to follow what happens
+with the proposal: updates on the proposal in reaction to comments,
+state of the proposal (when is it accepted, or rejected).
+
+The purpose of the TEP process is to reduce the amount of "tribal
+knowledge" in our community. This is done by putting in place a gate
+(submitting and getting a TEP merged) that mark a decision after
+having discussed the subject during video calls, on mailing list and
+other means. This process aims to enhance communication and
+discoverability. The TEP process is intended to create high quality
+uniform design and implementation documents for WGs to deliberate.
+
+A TEP is broken into sections which can be merged into source control
+incrementally in order to support an iterative development process. A
+number of section are required for a TEP to get merged in the
+`proposed` state (see the different states in the [TEP
+Metadata](#tep-metadata)). The rest of the section can be updated once
+being discussed more during Working Groups and agreed on.
+
+[road to Go 2]: https://blog.golang.org/toward-go2
+
+## Stewardship
+
+The following
+[DACI](https://en.wikipedia.org/wiki/Responsibility_assignment_matrix#DACI)
+model indentifies the responsible parties for TEPs:
+
+| **Workstream**          | **Driver**          | **Approver**             | **Contributor**                                      | **Informed** |
+| ---                     | ---                 | ---                      | ---                                                  | ---          |
+| TEP Process Stewardship | Tekton Contributors | Tekton Governing members | Tekton Contributors                                  | Community    |
+| Enhancement delivery    | Enhancement Owner   | Project(s) OWNERs        | Enhancement Implementer(s) (may overlap with Driver) | Community    |
+
+In a nutshell, this means:
+- Updates on the TEP process are driven by contributors and approved
+  by the tekton governing board.
+- Enhancement proposal are driven by contributors, and approved by the
+  related project(s) owners.
+
+## Reference-level explanation
+
+### What type of work should be tracked by a TEP
+
+The definition of what constitutes an "enhancement" is a foundational
+concern for the Tekton project. Roughly any Tekton user or operator
+facing enhancement should follow the TEP process. If an enhancement
+would be described in either written or verbal communication to anyone
+besides the TEP author or developer, then consider creating a
+TEP. This means any change that may impact any other community project
+in a way should be proposed as a TEP. Those changes can be for
+technical reason, adding or removing (deprecating then removing)
```

Comment:
Maybe reword this last sentence. Something like: Those changes could be for technical reasons, or adding new features, or deprecating then removing old features.